repos/system_upgrade/el7toel8/actors/systemfacts/libraries/systemfacts.py

```diff
@@ -212,18 +212,38 @@ def asbool(x):
     def identity(x):
         return x
 
+    def _get_config_file_values():
+        _outdata = {}
+        _key_mapping = (('SELINUX=', 'static_mode', identity),
+                       ('SELINUXTYPE=', 'policy', identity),)
+
+        _outdata['enabled'] = False
+        _outdata['mls_enabled'] = False
+
+        for item in call(['cat', '/etc/selinux/config']):
+            for (key, mapped, cast) in _key_mapping:
+                if item.startswith(key):
+                    _outdata[mapped] = cast(item[len(key):].strip())
+
+        return _outdata
```

Comment:
Information about `static_mode` and `policy` is taken from two sources of truth (`sestatus` in case selinux is enabled, config file in case selinux is disabled). I would prefer to use only one source of data (config file always) or maybe to setup `None` when data are not provided by the `sestatus` command? (@oamg/developers, guys what do you think? Another opinion is welcomed). Another think is that selinux can be disabled from the kernel commandline by command `selinux=0` - how to handle such situation? Probably topic for another PR (I prefer to resolve it separately as this is resolving the current problem we have now).

Reply:
We should definitely involve the selinux guys into this as there might be some special interactions among the various modes of enablement or disablement

Reply:
Two days ago on os-devel ML announcement from SELinux team introducing mtgs to discuss issues/questions/ideas. Too bad the first one is in late January.

Reply:
I guess that purpose of such mtgs is little different to the topic we are resolving here.

Reply:
Regarding SELinux related system facts gathering. I did a research on python "selinux" library.
I propose replacing current solution which is using combination of sestatus command and a /etc/selinux/config file reading with the above mentioned library usage which covers both runtime and static - config information.

enabled <- selinux.is_selinux_enabled()
mls_enabled <- selinux.is_selinux_mls_enabled()
runtime_mode <- selinux.security_getenforce()
static_mode <- selinux.selinux_getenforcemode()
policy <- selinux.selinux_getpolicytype()

Reply:
That solution looks good to me. The library is not part of the std python library, but it is part of the minimal installation setup and it is required by other important applications (like firewalld). So I think this would be ok to use those functions. What from the point of other guys? @oamg/developers